### ISSUE TITLE
? officially added to the Arduino library manager.?

### ISSUE THREAD

**Issue by TerryKing** [schinken/Flash#11]
Hi,
I have latest (1.6.13) IDE and the library manager does not who this library "added to the Arduino library manager."..  Am I missing something.

Direct install and examples are OK..

Thanls!

Regards, Terry King
...In The Woods in Vermont, USA
terry@yourduino.com

**Comment by mvhconsult** [comment]
In IDE 1.6.12 (Win) and 1.6.13 (Linux) it can be found in the library manager ("Manage Libraries"). Search for Flash, and several libraries pop up. Choose the one by 'Mikal Hart', and select version. 

You can check that it really is the one served from here by clicking "More info": you will be redirected to this github location.

Can you please check this works for you

PS: latest is 1.8.0 btw

**Comment by TerryKing** [comment]
SOLVED:  
I have Arduino IDE 1.6.12 and 1.6.13. 
In 1.6.12 and 1.6.13 Library Manager I see the Mikal Hart flash and 1.01 INSTALLED. Also see option of installing 1.0.0
I removed the Flash5 library I downloaded here. Installed the "Mikal Hart" 1.01 and it all works, compiles and runs.  
SO: My Confusion. Sorry. I thought the "Mikal Hart" flash was the original one, and not your updated version.  Maybe the title/info for that can show "1.8 or later fork"??  
THANKS! for the good work here...
Regards, Terry King
...In The Woods in Vermont, USA
terry@yourduino.com
